dbfit-java/build.gradle

```diff
@@ -290,10 +290,71 @@ project('teradata') {
 project('netezza') {
     description = 'DBFit netezza engine'
 
+    def propertiesFile = 'TestDbConnectionDbFitNetezza.properties'
+    def jdbcDriverClass = 'org.netezza.Driver'
+    def jdbcUrlPrefix = 'jdbc:netezza://'
+    def jdbcUrl
```

Comment:
I'm not quite sure about these kind of global variables (I'm not quite sure how many times and when are these initialized).